library/src/amd_detail/rocblaslt/src/include/tensile_host.hpp

```diff
@@ -232,13 +236,22 @@ struct RocblasltContractionProblem
         , scaleD(scaleD)
         , scaleE(scaleE)
         , scaleAlphaVec(scaleAlphaVec)
+        , isScaleAVec(isScaleAVec)
+        , isScaleBVec(isScaleBVec)
         , bias_type(bias_type)
         , epilogue(epilogue)
         , workspace(workspace)
         , workspaceSize(workspaceSize)
         , stream(stream)
         , Synchronizer(Synchronizer)
     {
+        if(isScaleAVec != isScaleBVec)
+        {
+            // If one of the scales is a vector, the other must be as well
+            // we turn it off automatically for now to avoid crash.
+            isScaleAVec = false;
+            isScaleBVec = false;
```

Comment:
Should we return error instead off forcing turn it off?

Reply:
I'll have to move it to auxiliary because this is inside a constructor.